src/Database/Migrations/DicomAdapterContextModelSnapshot.cs

```diff
@@ -0,0 +1,158 @@
+﻿// <auto-generated />
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using Nvidia.Clara.DicomAdapter.Database;
+
+namespace Nvidia.Clara.DicomAdapter.Database.Migrations
+{
+    [DbContext(typeof(DicomAdapterContext))]
+    partial class DicomAdapterContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
+            modelBuilder
+                .HasAnnotation("ProductVersion", "3.1.11");
```

Comment:
seems like a lot of these values should be constants somewhere -- what happens when they need to be updated?

if they're only ever referenced here, I guess not having constant is fine; but if the value is duplicated in code, let's not do that

Reply:
Generated code.  When there's a change to db schema, we'll run `dotnet ef migrate` to generate updates.